xtrx.c

```diff
@@ -27,6 +27,9 @@
 #include <linux/kdev_t.h>
 #include <linux/fs.h>
 #include <linux/cdev.h>
+#ifdef CONFIG_CPU_RK3399
```

Comment:
Is this related to the patch or an accidental?

If it's related - please explain the reasoning in the commit message, or split into separate commits.

Reply:
it's accidental (but without this fix driver won't build on rockpro and CM4)

Reply:
Thank you. Please submit the CONFIG_CPU_RK3399 fix separately.